ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/validation/PrettyValidationErrors.scala

```diff
@@ -48,7 +48,6 @@ object PrettyValidationErrors {
       case InvalidVariableOutputName(varName, _, paramName) => node(s"Variable output name '$varName' contains unsupported chars.", "Please use only letters, numbers or '_'.", fieldName = paramName)
       case NotSupportedExpressionLanguage(languageId, _) => node(s"Language $languageId is not supported", "Currently only SPEL expressions are supported")
       case MissingPart(id) => node("MissingPart", s"Node $id has missing part")
-      case WrongProcessType() => node("Wrong scenario type", "Scenario type doesn't match category - please check configuration")
```

Comment:
WrongProcessType is present in Errors.md, let's remove there and move Errors.md to /docs or remove Errors.md completelty